src/checkin/CheckInQueue.jsx

```diff
@@ -122,7 +112,7 @@ class CheckInQueue extends React.Component {
           ] }
           rowData={ Object.values(this.props.patients) }
           rowSelectedActionCreators={[() => screeningActions.setLastScreeningQueue(this.props.screeningLocation), patientActions.setSelectedPatient, this.redirectToScreeningTaskList.bind(this)]}
-          searchFilterFields={['name.fullName']}
+          searchFilterFields={['name.fullName', 'name.reverseFullName']}
```

Comment:
I was thinking that this line would work, but maybe I was wrong:

`searchFilterFields={[ (patient) => patient.familyName + " " + patient.givenName + " " + patient.familyName ]}`

Reply:
Yes I tried it, didn't work.
I believe the package mostly treats in as a path to the item in the object. So those concatenation operations would only  apply on the path. I.e the function above would return `["patient.familyName patient.givenName patient.familyName"]` but that's not a path to any value in the patient object.